packages/e2e-test-utils/src/find-sidebar-panel-with-title.js

```diff
@@ -3,12 +3,14 @@
  */
 import { first } from 'lodash';
 
+/** @typedef {import('./puppeteer').ElementHandle} ElementHandle */
```

Comment:
Is this meant to be a relative path? Or should it be `import('puppeteer')` ?

I think once we have these [files included in `tsconfig.json`](https://github.com/WordPress/gutenberg/blob/d2188955efe542d5c8a26bb2b1d07983088c4524/tsconfig.json#L24-L26), we should start to get errors for these issues.

![image](https://user-images.githubusercontent.com/1779930/69892701-56fddb00-12d6-11ea-933c-fada269ffcf8.png)

```suggestion
/** @typedef {import('puppeteer').ElementHandle} ElementHandle */
```

Technically the above won't work correctly either, at least not without also installing [`@types/puppeteer`](https://www.npmjs.com/package/@types/puppeteer), since Puppeteer doesn't publish its own types.

Fun fact: By temporarily adding this file to the `includes` set, it highlighted a legitimate error in our `@return` statement a few lines ago. Since the function is `async`, it doesn't return an `ElementHandle`, it returns a `Promise<ElementHandle>`.

Reply:
It was a typo, the part which references Puppeteer. Can you push changes directly to this branch? I’m still learning all that stuff. My very first task for next week is to migrate to Visual Studio Code 😜

Reply:
I fixed imports for both `puppeteer` and `moment`.

Can we handle the individual issues as we enabled TS validation?